src/main/java/commands/WorkoutCommand.java

```diff
@@ -165,7 +165,8 @@ public void execute() {
                 String currentWorkout = getWorkoutList().getCurrentWorkout(getUserArguments());
                 Workout updatedWorkout = getWorkoutList().updateWorkout(getUserArguments());
                 getUI().printUpdateWorkoutMessage(currentWorkout, updatedWorkout);
-                getPlanList().updatePlanContainsUpdatedWorkout(currentWorkout, updatedWorkout);
+                getPlanList().updatePlanContainsUpdatedWorkout(currentWorkout,
+                        updatedWorkout.toString(), updatedWorkout);
```

Comment:
just wondering, is there a reason why the 2nd parameter is added? I feel like the toString method on updatedWorkout can be called in their respective methods x)?

Reply:
is this new method only serving for update plan, and the delete plan will use the other method with two parameters?

Reply:
Because the original updatePlanContainsUpdatedWorkout() is shared with delete feature, and delete feature will not have a updatedWorkout

Reply:
My bad, now I look again and I feel you are right, will make changes regarding it, thanks!